src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -554,25 +633,21 @@ protected FieldValueInfo completeValueForList(ExecutionContext executionContext,
             index++;
         }
 
-        CompletableFuture<List<ExecutionResult>> resultsFuture = Async.each(fieldValueInfos, (item, i) -> item.getFieldValue());
+        CompletableFuture<List<Object>> resultsFuture = Async.each(fieldValueInfos, (item, i) -> item.getValue());
 
-        CompletableFuture<ExecutionResult> overallResult = new CompletableFuture<>();
-        completeListCtx.onDispatched(overallResult);
+        CompletableFuture<Object> overallResult = new CompletableFuture<>();
+        //completeListCtx.onDispatched(overallResult);
```

Comment:
I commented out here but its needs to change the `Instrumentation` method signatures from `ER` to `Object`